src/widget/form/groupchatform.cpp

```diff
@@ -367,5 +371,9 @@ void GroupChatForm::keyReleaseEvent(QKeyEvent* ev)
 
 void GroupChatForm::retranslateUi()
 {
-    nusersLabel->setText(GroupChatForm::tr("%1 users in chat", "Number of users in chat").arg(group->getPeersCount()));
```

Comment:
Why is there `GroupChatForm::tr` instead of `tr` ?
The `tr` is used in a `GroupChatForm` method.